docker/Makefile

```diff
@@ -0,0 +1,18 @@
+build:
+	@docker build -t scout:latest ../
+	@docker build -t scout-setup:v1.0 scout-setup/
+	@docker build -t scout-import:v1.0 scout-import/
```

Comment:
Is it necessary to have one container for each possible CLI command? Maybe all CLI stuff could be done using a single image, it seems the only thing that differs between them is the entrypoint.

Reply:
It can definitely be done in a single image, just that image will be very complex. The more commands you add the more complex the image will be. But I agree, this setup might not be optimal either. If you have a good solution please feel free to change

Reply:
Hi @ninanorgren I'm not really fond of this solution with different containers for different CLI commands. I'll to write a simpler Dockerfile for handling both the CLI and the server side of Scout, with suggestions on how to connect it to other services, like mongodb and nginx.